services/workflows-service/src/workflow/workflow.controller.internal.unit.test.ts

```diff
@@ -63,6 +63,8 @@ function buildWorkflowDeifintion(sequenceNum) {
       type: 'json-schema',
       schema: {},
     },
+    projectId: projectId,
+    isPublic: !projectId,
```

Comment:
This seems like it could lead to confusion or flaky tests. Myabe `isPublic` should be an argument?